### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by bru32** [gmnevton/JSON.VerySimple#1]
A few examples would be worth adding (especially for adding an Array).
I want to add a child node

    node := obj.AddChild('stats', jtArray);

and then set the value

    node.Value := '[1,2,3,4,5]';

But this doesn't work.  Instead, I'm having to do this...

    v := node.AddChild('', jtNumber); v.Value := '1';
    v := node.AddChild('', jtNumber); v.Value := '2';
    v := node.AddChild('', jtNumber); v.Value := '3';

**Comment by gmnevton** [comment]
Ok. But later.

If we are on this topic, than You must remember this fiew things about JSON.VerySimple:
{ - means object,
[ - means array,

and node.Value holds one value of type: jtString, jtNumber, jtTrue, jtFalse, jtNull.
Nodes of type  jtObject, jtArray can't have any value, they are parent nodes.
You must create some child nodes or leave them empty.

So, Your use case is correct:
> v := node.AddChild('', jtNumber); v.Value := '1';
> v := node.AddChild('', jtNumber); v.Value := '2';
> v := node.AddChild('', jtNumber); v.Value := '3';

You can shorten this to:
node.AddChild('', jtNumber).Value := '1';
node.AddChild('', jtNumber).Value := '2';
node.AddChild('', jtNumber).Value := '3';

And last thing, JSON.VerySimple supports json that starts from object or array node.

For egzample: https://www.w3schools.com/js/js_json_intro.asp

{"name":"John", "age":30, "car":null}

or

{"employees":[
  { "firstName":"John", "lastName":"Doe" },
  { "firstName":"Anna", "lastName":"Smith" },
  { "firstName":"Peter", "lastName":"Jones" }
]}

or

{
"name":"John",
"age":30,
"cars":["Ford", "BMW", "Fiat"]
}

or

[
"1",
"some object": {
  "some key1": "some value 1",
  "some key2": "some value 2"
},
"2"
]

I uploaded en update to JSON.VerySimple. I forgot that I made some changes while ago. Enjoy.

**Comment by gmnevton** [comment]
I will make some examples, but for now i close this issue.
